src/plot_erpgrid.jl

```diff
@@ -29,25 +28,26 @@ function plot_erpgrid!(
     times = -1:size(data, 2)-2, #arbitrary strat just for fun
     kwargs...,
 )
-    chanNum = size(data, 1)
-    data = data[1:chanNum, :]
+    config = PlotConfig(:erpgrid)
+    config_kwargs!(config; kwargs...)
+
+    chan_num = size(data, 1)
+    data = data[1:chan_num, :]
     pos = hcat([[p[1], p[2]] for p in pos]...)
 
-    pos = pos[:, 1:chanNum]
+    pos = pos[:, 1:chan_num]
     minmaxrange = (maximum(pos, dims = 2) - minimum(pos, dims = 2))
     pos = (pos .- mean(pos, dims = 2)) ./ minmaxrange .+ 0.5
 
     axlist = []
-    #ax = Axis(f[1, 1],backgroundcolor=:green)#
-
     rel_zeropoint = argmin(abs.(times)) ./ length(times)
 
     for (ix, p) in enumerate(eachcol(pos))
         x = p[1] #- 0.1
         y = p[2] #- 0.1
         # todo: 0.1 should go into plot config
         ax = Axis(
-            f[1, 1],
+            f[1:8, 1:8],
```

Comment:
I dont understand why this is necessary - 1:8 seems arbitrary?

Reply:
trays of old solution, I will remove it